.stylelintrc.json

```diff
@@ -1,7 +1,6 @@
 {
   "extends": [
-    "stylelint-config-recommended-scss",
-    "stylelint-prettier/recommended"
+    "@18f/identity-stylelint-config"
```

Comment:
I've updated to login.gov's stylelint config, as it simplifies conforming to TTS CSS/Sass coding standards